_posts/2020-04-16-foodsearchinc.md

```diff
@@ -0,0 +1,51 @@
+
+**Describe your project?**
+
+Our web application [foodsearchinc.com](https://foodsearchinc.com/), is a directory that makes it easier for small businesses like restaurants, grocers, bakeries, and anyone in the food industry to continue to reach their customers by indicating if they offer pickup or delivery services. Our project, while focused on Edmonton and area, can be scaled to any city. Data is solicited through google forms, verified by businesses,  and then uploaded to a database. User accounts will be available for businesses to manage their own data in the near future.
+
+ Our project will be expanded to allow businesses to list menu items on a per dish basis, and provide more information about their amazing food. With this information, customers can create more targeted filtering and searches for the food they want, where to find it, and eventually order dishes directly from each place for pickup.
+  
+**How did your team come together?**
+
+Our company, Food Search Inc., was created to increase the quality of information, and facilitate access to the food we eat, by creating a database of products and dishes with information such as:  ingredients, allergens, and link this to information about where to find it. In January 2020, [Tyler](https://www.linkedin.com/in/tylerfindlay/) and I started our first project [searchglutenfree.com](https://searchglutenfree.com/), which helps individuals with celiac disease or gluten intolerance, find restaurants and dishes much easier. Due to the pandemic, we decided to repurpose the code, and created foodsearchinc.com to address problems faced by the food industry. Our team was made possible through networking opportunities facilitated by [Startup Edmonton](https://www.startupedmonton.com/ ). A few chilled beers and meetings later, we teamed up to change the way people find information about the food they eat.
+
+**Where is your team based?**
+
+Tyler Findlay and I are from Edmonton.
+
+**Can you describe the skills you bring to the project?**
+
+Tyler, the technical co-founder is a full-stack software developer and scrum master. I am a non-technical co-founder, and manage the operational and business development side.
+
+**How does it feel to be collaborating on a project at a time like this?**
+
+Rewarding, and solving a problem in real time while speaking with over 400 restaurant owners has been fulfilling. 
+
+**What tools, made available by the  hackathon, are you using?**
+
+[*Slack*](https://app.slack.com/client/TV5MXDY8Z/D012J7RHR7A), and [*FindCollabs*](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed)
+
+**Is there anything else you’d like to say?**
+
+Thank you to the team that put Codevid-19 together! It’s such an amazing initiative for creating projects to help out during these uncertain times.
+
+Thank you from Codevid-19 for your fine effort, and best of luck in the future.
```

Comment:
CODEVID-19

Reply:
Bump

Reply:
qu'est-ce que tu sais "Bump" If that's slang, don't forget that I'm old.

Reply:
Same here, see original comment